packages/elements/src/number-field/index.ts

```diff
@@ -532,6 +532,29 @@ export class NumberField extends FormFieldElement {
     return 0;
   }
 
+  /**
+   * Count precision number
+   * @param number value to count
+   * @returns precision number
+   */
+  private getPrecision (number: number): number {
+    const getDecimalPrecision = (number: string | number): number => {
+      const [whole, part] = number.toString().split('.');
```

Comment:
Would this make it a bit more readable ?
```suggestion
      const [wholeNumber, decimalNumber] = number.toString().split('.');
```